src/GraphQL/Types/Scalars/TimeSpanMillisecondsGraphType.cs

```diff
@@ -41,10 +34,18 @@ public TimeSpanMillisecondsGraphType()
             TimeSpan _ => value, // no boxing
             int i => TimeSpan.FromMilliseconds(i),
             long l => TimeSpan.FromMilliseconds(l),
-            _ => null
+            null => null,
```

Comment:
Add other integer cases ?

Reply:
Ok

Reply:
Done.  Note that `FromMilliseconds` and `FromSeconds` accept a `double`.  We could also support floating-point values here.  I did not do so yet.

Reply:
Yes, **internal** representations of these scalars accept but not external format. Let's keep it as integer values.